.changeset/purple-kids-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-js': minor
```

Comment:
I think this is more of patch than a minor.

Reply:
updated